### DISCUSSION TITLE
Missing pycodestyle's `convention` option

### DISCUSSION THREAD

**Discussion by ghost** [astral-sh/ruff#10156] [Q&A] [answered]
Hello! I want to use `ruff` instead of `flake8`, but it seems that I can't configure the docstring convention. This option is [documented on the website](https://docs.astral.sh/ruff/settings/#lint_pydocstyle_convention), but is missing in 0.2.2 (downloaded from PyPi, if that helps).
```toml
preview = true

[lint.pycodestyle]
convention = "google"
```
```log
$ ruff test/test.py
ruff failed
  Cause: Failed to parse ~/ruff.toml
  Cause: TOML parse error at line 3, column 2
  |
3 | [lint.pycodestyle]
  |  ^^^^
unknown field `convention`, expected one of `max-line-length`, `max-doc-length`, `ignore-overlong-task-comments`

**Comment by charliermarsh** [comment, marked as answer]
Hey sorry -- I believe it's `[lint.pydocstyle]`, rather than `[lint.pycodestyle]`. Can you try that?

**Reply by ghost** [reply]
Thank you 😄
